jta-1_2-standalone/src/test/java/org/jboss/narayana/quickstarts/jta/TestCase.java

```diff
@@ -50,8 +45,6 @@
 
     private MandatoryCounterManager mandatoryCounterManager;
 
-    private TestEntityRepository testEntityRepository;
-
     private Weld weld;
 
     @BeforeClass
```

Comment:
It executes the example with the help of the unit test. We have a lot of quickstarts like that.